.github/workflows/lint.yml

```diff
@@ -0,0 +1,17 @@
+name: Lint
+
+on: push
```

Comment:
This should probably run for pull requests as well, shouldn't it?

Reply:
Added `pull_request` for both workflows